static/app/views/alerts/rules/details/body.tsx

```diff
@@ -409,6 +423,10 @@ const StyledLayoutBody = styled(Layout.Body)`
   }
 `;
 
+const StyledLayoutBodyWrapper = styled(Layout.Body)`
+  margin-bottom: -${space(3)};
+`;
+
```

Comment:
This is the footer alignment hack (I'll get round to fixing this properly at some point).